Tentacle/Issue.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  Issue.swift
+//  Tentacle
+//
+//  Created by Romain Pouclet on 2016-05-23.
+//  Copyright © 2016 Matt Diephouse. All rights reserved.
+//
+
+import Argo
+import Curry
+
+/// An Issue on Github
+public struct Issue: Hashable, CustomStringConvertible {
+    public enum State: String {
+		case Open = "open"
+        case Closed = "closed"
+	}
+	
+    public let id: Int
+    public let url: NSURL?
+
+	public let number: Int
+	public let state: State
+	public let title: String
+	public let body: String
+	public let user: User?
+	public let labels: [Label]
+	public let assignee: User?
+	public let milestone: Milestone?
+	public let locked: Bool
+	public let comments: Int
+    public let pullRequest: PullRequest?
+	public let closedAt: NSDate?
+	public let createdAt: NSDate?
+	public let updatedAt: NSDate?
```

Comment:
I don't think `createdAt` or `updatedAt` can be `nil`, so they shouldn't be `Optional`..